front/src/modules/rollingStock/hooks/useFilterRollingStock.tsx

```diff
@@ -129,8 +129,14 @@ export function computeFilter(filter: RollingStockFilterKeys, filters: RollingSt
   };
 }
 
-export default function useFilterRollingStock() {
+type RollingStockFetchArgs = {
+  freightCompatible?: boolean;
+  pageSize?: number;
```

Comment:
Why only `pageSize`? Maybe you also want to add `page` (see [`PaginationQueryParam`](https://github.com/OpenRailAssociation/osrd/blob/68e0513349a18e7c9faeed6724919ec161fcb046/editoast/src/views/pagination.rs#L127-L133)).